index.html

```diff
@@ -73,6 +76,196 @@ <h3 class="social_title blue">LET’S CONNECT</h3>
+              sign-ups required.
+            </p>
+            <ul class="work_cats">
+              <li>Html</li>
+              <li>Css</li>
+              <li>Javascript</li>
+            </ul>
+            <button type="button" class="default_btn">See More</button>
+          </div>
+        </div>
+      </section>
+      <!-- end of work section-->
+
+      <section class="about">
+        <h1 class="headline_title">About Myself</h1>
+        <p class="about_details">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <h3 class="social_title blue">LET’S CONNECT</h3>
+        <ul class="social_links">
+          <li class="social_link">
+            <a href="#"
+              ><img
+                class="social_icon"
+                src="./images/twitter.svg"
+                alt="Twitter"
+            /></a>
+          </li>
+          <li class="social_link">
+            <a href="#"
+              ><img
+                class="social_icon"
+                src="./images/linkedin-icon.svg"
+                alt="Linkedin"
+            /></a>
+          </li>
+          <li class="social_link">
+            <a href="#"
+              ><img class="social_icon" src="./images/m.svg" alt="M"
+            /></a>
+          </li>
+          <li class="social_link">
+            <a href="#"
+              ><img class="social_icon" src="./images/github.svg" alt="Github"
+            /></a>
+          </li>
+          <li class="social_link">
+            <a href="#"
+              ><img class="social_icon" src="./images/doit.svg" alt="Diot"
+            /></a>
+          </li>
+        </ul>
+        <button type="button" class="default_btn">Get My Resume</button>
+        <ul class="about_ul">
+          <h2 class="sub_title">
+            Languages
+            <img
+              class="arrow_icon"
+              src="./images/down-arrow-icon.png"
+              alt="down"
+            />
+          </h2>
+          <li class="ul_item">
+            <img src="./images/js-icon.png" alt="Javascript" /><span
+              >Javascript</span
+            >
+          </li>
+          <li class="ul_item">
+            <img src="./images/html-icon.png" alt="HTML" /><span>HTML</span>
+          </li>
+          <li class="ul_item">
+            <img src="./images/css-icon.png" alt="CSS" /><span>CSS</span>
+          </li>
+        </ul>
+        <ul class="about_ul">
+          <h2 class="sub_title">
+            Frameworks
+            <img
+              class="arrow_icon"
+              src="./images/right-arrow-icon.png"
+              alt="down"
+            />
+          </h2>
+        </ul>
+        <ul class="about_ul">
+          <h2 class="sub_title">
```

Comment:
- [ ] Kindly make sure you stick to the design 
`Hint: You can use flexbox to place these elements here.`
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/198303356-0f846e5e-0765-4777-a2d2-f51b4c79b622.png)

Figma temaplet :point_down: 
![image](https://user-images.githubusercontent.com/97100091/198303221-6f1421fa-f008-4c60-ae6e-8ea04fb99f5a.png)